plugins/fresco-pd/fresco-pd.quirk

```diff
@@ -0,0 +1,7 @@
+# FL7102
+[Guid=USB\VID_1D5C&PID_7102]
+Plugin = fresco_pd
+
+# FL7112
+[Guid=USB\VID_1D5C&PID_7112]
+Plugin = fresco_pd
```

Comment:
Any particular reason not to just match everything that is `1D5C` vendor?  Do they make other devices too?

Reply:
I believe other kinds of device from Fresco have a different protocol. Apparently a FrescoPd plugin was appropriate.

Reply:
OK